src/classes/EventHandlers.js

```diff
@@ -767,10 +773,10 @@ class EventHandlers {
 	 * @param  {number} 	app_id			tracker id
 	 * @param  {number} 	cat_id			tracker category id
 	 * @param  {number} 	tab_id			tab id
-	 * @param  {string} tab_host		tab url host
-	 * @param  {string} page_url		full tab url
+	 * @param  {string} 	tab_host		tab url host
+	 * @param  {string} 	page_url		full tab url
 	 * @param  {number} 	request_id		request id
-	 * @return {boolean}
+	 * @return {Object}		{block, ss_unblock}
 	 */
 	_checkBlocking(app_id, cat_id, tab_id, tab_host, page_url, request_id) {
```

Comment:
this should have an object return, correct? looks to just be returning `block` at the moment, instead of `return {block:true, ss_unblock:false}` etc

See line 793

Reply:
block IS object, as it is returned by shouldBlock, which returns object.